sphinx_needs/lsp/esbonio.py

```diff
@@ -0,0 +1,385 @@
+"""Suport Sphinx-Needs language features."""
+import getpass
+import os
+import pathlib
+import re
+from hashlib import blake2b
+from typing import List, Optional, Tuple, Union
+
+from esbonio.lsp import LanguageFeature
+from esbonio.lsp.rst import CompletionContext, DefinitionContext, HoverContext
+from esbonio.lsp.sphinx import SphinxLanguageServer
+from pygls.lsp.types import (
+    CompletionItem,
+    CompletionItemKind,
+    InsertTextFormat,
+    Location,
+    Position,
+    Range,
+    TextEdit,
+)
+
+from sphinx_needs.lsp.needs_store import NeedsStore
+
+
+class NeedlsFeatures(LanguageFeature):  # type: ignore[misc]
+    """Sphinx-Needs features support for the language server."""
+
+    def __init__(self, *args, **kwargs) -> None:  # type: ignore[no-untyped-def]
+        super().__init__(*args, **kwargs)
```

Comment:
is there any reason to use `args, **kwargs)` here? the arguments for `super().__init__(...)` are well-defined.

https://github.com/swyddfa/esbonio/blob/6d898fc6ad691bf1d1d63ec496c6c453b5699fa2/lib/esbonio/esbonio/lsp/rst/__init__.py#L227 

then you can remove the `type: ignore` comment here